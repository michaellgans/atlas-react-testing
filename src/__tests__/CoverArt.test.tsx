// Test for CoverArt Component

// Imports
import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import { CoverArt } from "@/components/CoverArt";
import { ContextProvider } from "@/components/ContextProvider";

test("CoverArt renders correctly", () => {
    const {container} = render(
        <ContextProvider>
            <CoverArt />
        </ContextProvider>
    );
    expect(container).toMatchSnapshot();
});
