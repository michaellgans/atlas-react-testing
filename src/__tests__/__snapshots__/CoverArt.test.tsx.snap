// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CoverArt renders correctly 1`] = `
<div>
  <div
    class="cover-art mb-6"
  >
    <img
      alt="Placeholder for Loading"
      class="image-skeleton animate-pulse rounded-md"
      src="/src/assets/placeholder.svg"
    />
  </div>
</div>
`;
